pkg/database/migrations.go

```diff
@@ -236,29 +217,19 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				}
 
 				// Add realm relations to the rest of the tables.
-				logger.Debugw("adding RealmID to users")
 				if err := tx.AutoMigrate(&User{}).Error; err != nil {
 					return err
 				}
-				logger.Debugw("joining users to the default realm")
 				var users []*User
 				if err := tx.Find(&users).Error; err != nil {
 					return err
 				}
 				for _, u := range users {
-					logger.Debugw("adding user to default realm", "user", u.ID)
-
-					u.AddRealm(defaultRealm)
-					if u.SystemAdmin {
-						u.AddRealmAdmin(defaultRealm)
-					}
-
-					if err := tx.Save(u).Error; err != nil {
+					if err := u.AddToRealm(db, defaultRealm, rbac.LegacyRealmAdmin, System); err != nil {
```

Comment:
This migration is semantically different than it was before

Reply:
Done